egs/mgb2_arabic/s5/local/chain/tuning/run_tdnn_lstm_1a_disc.sh

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+# note: this relies on having a cluster that has plenty of CPUs as well as GPUs,
+# since the alignment and the lattice generation/egs-dumping takes quite a bit
+# of CPU time.
+
```

Comment:
Is this script actually being used?  The WERs in the comments seem like they might be outdated.  How much improvement are you getting from this, if you are using it?

Reply:
This gave 2% WER improvement with a previous run of the experiment. I have not run and tested this recently.